### DISCUSSION TITLE
Training Recognition models within  Macbook GPUs

### DISCUSSION THREAD

**Discussion by kasuba-badri-vishal** [mindee/doctr#1168] [Q&A]
Hi, 
For DocTR Recognition models training, the GPU availaiblity is based on using CUDA. But in Mac M1 chip based GPUs , there are mps devices to using GPUs and I am not able to train my DocTR models using mps devices, instead had to work on CPUs only. 

Please kindly let me know how could I train recognition models using GPU in MAC M1

**Comment by felixdittrich92** [comment]
Hey @kasuba-badri-vishal :wave:
Have you tried the additional steps from the documentation ? :)
https://github.com/mindee/doctr/pull/1076/files
